### PR TITLE
[58] calendar page

### PR DESCRIPTION
**Pull request by ozhurova**
added calendar page 
image

### CODE REVIEW

File: src/app/pages/calendar-page/calendar-page.component.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@import '../../shared/constants/colors.scss';
+@import '../../shared/constants/otherStyles.scss';
+
+  .fc { /* the calendar root */
+    max-width: $width-content;
+    max-height: 100vh;
+    margin: 0 auto;
+
+    padding: 20px;
+    background-color: #FAFAFA;
```
Comment (mhasilau) [inline src/app/pages/calendar-page/calendar-page.component.scss:10]: change to constant color

File: src/app/pages/calendar-page/calendar-page.component.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  currentEvents: EventApi[] = [];
+
+  handleWeekendsToggle(): void {
+    const { calendarOptions } = this;
+    calendarOptions.weekends = !calendarOptions.weekends;
+  }
+
+  handleDateSelect(selectInfo: DateSelectArg): void {
+    this.openModal();
+    // const title = prompt('Please enter a new title for your event');
```
Comment (mhasilau) [inline src/app/pages/calendar-page/calendar-page.component.ts:40]: pls del comment

File: src/app/pages/calendar-page/calendar-page.component.scss (hunk 1 above)
Comment (ozhurova) [reply src/app/pages/calendar-page/calendar-page.component.scss]: done

File: src/app/pages/calendar-page/calendar-page.component.ts (hunk 2 above)
Comment (ozhurova) [reply src/app/pages/calendar-page/calendar-page.component.ts]: done

**Review by vstrabko — APPROVED** [review]

**Review by mhasilau — APPROVED** [review]

**Review by PaulRomanov — APPROVED** [review]

**Pull request merged**